tests/unit/test_prometheus_schema_contract.py

```
@@ -197,8 +200,13 @@ def test_frozen_label_keys_for_core_metrics(render_metrics):
         "aeo1_exit_dns_latency_ms": {"fingerprint", "familyid"},
         "aeo1_exit_dns_consecutive_failures": {"fingerprint", "familyid"},
         "aeo1_exit_relay_info": {"fingerprint", "familyid", "nick", "verifiedaroi"},
-        "aeo1_aroi_relay_state": {"fingerprint", "familyid", "state"},
+        # B7.1: aeo1_aroi_relay_state labelset extended with bounded
+        # ciissversion + proof_type_family labels (cardinality cap = 4×5=20).
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)